### DISCUSSION TITLE
Withdrawn update

### DISCUSSION THREAD

**Discussion by ChristianKraemer** [Homebrew/discussions#4121] [Casks]
### Output of `brew config`

```shell
does not matter
```

### Output of `brew doctor`

```shell
does not matter
```

### Description of issue

A few hours ago I created a pull request to update nextcloud to v. 3.6.5 (https://github.com/Homebrew/homebrew-cask/commit/a9bbc12c6137161a7f95da7aa51acf4e357d5372). Now this update is withdrawn, fortunately not deleted, "only" marked as pre-release. Probably not a big problem, only nasty.
Is there anything I can do at the moment to revert my commit?

Greetings Christian 

### Relevant casks

nextcloud.rb

**Comment by SMillerDev** [comment]
You could make a pull request that downgrades it again

**Comment by ChristianKraemer** [comment]
Tried it. Failed with:
```
Error: Failure while executing; `git push --set-upstream https://******@github.com/ChristianKraemer/homebrew-cask.git bump-nextcloud-3.6.4:bump-nextcloud-3.6.4` exited with 1. Here's the output:
To https://github.com/ChristianKraemer/homebrew-cask.git
 ! [rejected]              bump-nextcloud-3.6.4 -> bump-nextcloud-3.6.4 (fetch first)
error: failed to push some refs to 'https://github.com/ChristianKraemer/homebrew-cask.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```
Greetings Christian

**Reply by SMillerDev** [reply]
I think if you remove your existing branch this should be fixed

**Comment by ChristianKraemer** [comment]
This error message appeared the first time I tried it. But my repo was old, created by cask-repair, so maybe configuration was different to the settings created by brew itself.
Anyway, I deleted the whole repo (with sweating hands :-) and the PR (https://github.com/Homebrew/homebrew-cask/pull/139780) is on its way.

Greetings Christian
